charts/datahub/subcharts/datahub-mce-consumer/templates/deployment.yaml

```diff
@@ -101,10 +101,64 @@ spec:
               value: "{{ . }}"
             {{- end }}
             {{- end }}
+            # TODO: Remove DATAHUB_GMS_HOST/PORT after v0.9.6
             - name: DATAHUB_GMS_HOST
               value: {{ printf "%s-%s" .Release.Name "datahub-gms" }}
             - name: DATAHUB_GMS_PORT
               value: "{{ .Values.global.datahub.gms.port }}"
+            - name: ENTITY_REGISTRY_CONFIG_PATH
+              value: /datahub/datahub-mce-consumer/resources/entity-registry.yml
+            - name: EBEAN_DATASOURCE_USERNAME
+              value: "{{ .Values.global.sql.datasource.username }}"
+            - name: EBEAN_DATASOURCE_PASSWORD
```

Comment:
we support password as value for other resources, we should do it here too for consistency

Reply:
like here: https://github.com/acryldata/datahub-helm/blob/master/charts/datahub/templates/datahub-upgrade/_upgrade.tpl#L20